### DISCUSSION TITLE
General Discussions!

### DISCUSSION THREAD

**Discussion by pssolanki111** [pssolanki111/polygon#1] [General]
Please use this discussion Thread for all **general conversations**, **questions** about the project or library, **Ideas** or **requests**...

Note that this is not meant for filing bugs. Use the [Issue Tracker](https://github.com/pssolanki111/polygon/issues) if you have a bug to submit.

**Comment by pwndizzle** [comment]
hey, is it possible to do a v1.1.1 release to add the as_of changes and deploy to pypi?

**Comment by pssolanki111** [comment]
hey @pwndizzle 

yes. I plan to make a new release. I joined a full time role which limited my availability. Looking to get back to the previous pace again wrt the lib. 

If you are able to make contributions of any size, that's appreciated as well. 

Adding `as_of` support should be a fairly simple PR.

**Reply by pwndizzle** [reply]
I think you previously merged the as_of changes, so the code is done, it was just that a release was never completed. I can make the release PR if that would help?

**Reply by pssolanki111** [reply]
wait what 👀 
Which commit are you referring to? @pwndizzle 

I can make a release today lol

**Reply by pssolanki111** [reply]
ahh I see it. https://github.com/pssolanki111/polygon/commit/fa491ad78815e46334f75759cd217bcf9480b451

**Reply by pwndizzle** [reply]
yep that's the one.

**Reply by pssolanki111** [reply]
Awesome. Creating a release now. I just need to set up my dev env again (my machine changed in the process). I'll update here once release is done

**Reply by pssolanki111** [reply]
@pwndizzle Release 1.1.1 is on PyPi. 

Thanks for bringing this to my attention. Cheers!

**Reply by pwndizzle** [reply]
Thanks for the fast turn around!
